README.md

```diff
@@ -27,4 +31,48 @@ val newAccountKeypair = wallet.create()
 
 // Addresses are shortened for example only
 // publicKey=GB7S...GETPQ, secretKey=SBTL...GI27
+```
+
+### `Wallet.fund()`
+
+```kotlin
+fun fund(
+    sourceAddress: String,
+    destinationAddress: String,
+    startingBalance: String = "1",
+    sponsorAddress: String = ""
+): Transaction
+```
+
+Example
+
+```kotlin
+val sourceAddress = "GAMQTINWD3YPP3GLTQZ4M6FKCCSRGROQLIIRVECIFC6VEGL5F64CND22"
+val destinationAddress = "GDEIYYWIVK24CCQ3Y4QNGEIBEFABTTCFBRTVNQZ43VPOUNQARO7ZEKJY"
+
+val sponsoredFundTransaction =
+    wallet.fund(sourceAddress, destinationAddress, sponsorAddress = sourceAddress)
+```
+
+## Utils
+
+### `Operation.createSponsoredOperation()`
```

Comment:
Kotlin API (like Java API) docs are usually done in [KDoc](https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax). If it is not necessary to expose these functions in README, it is easier to do it in KDoc.